setup.cfg

```diff
@@ -1,5 +1,77 @@
+[metadata]
+name = pylint
+version = attr: pylint.__version__
+description = python code static checker
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/PyCQA/pylint
+author = Python Code Quality Authority
+author_email = code-quality@python.org
+license = GPL-2.0-or-later
+license_files =
+    COPYING
+    CONTRIBUTORS.txt
+classifiers =
+    Development Status :: 6 - Mature
+    Environment :: Console
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License (GPL)
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: Implementation :: CPython
+    Programming Language :: Python :: Implementation :: PyPy
+    Topic :: Software Development :: Debuggers
+    Topic :: Software Development :: Quality Assurance
+    Topic :: Software Development :: Testing
+keywords = static code analysis linter python lint
+project_urls =
+    "What's New" = https://pylint.pycqa.org/en/latest/whatsnew/
+    "Bug tracker" = https://github.com/PyCQA/pylint/issues
+    "Discord server" = https://discord.gg/kFebW799
+    "User manual" = http://pylint.pycqa.org/en/latest/
+    "Contributing" = http://pylint.pycqa.org/en/latest/development_guide/contribute.html
+    "Technical references" = http://pylint.pycqa.org/en/latest/technical_reference/index.html
+    "Mailing list" = mailto:code-quality@python.org
+
+[options]
+packages = find:
```

Comment:
I think, during the rebase, you removed the `[options.packages.find]` section. Or is there another reason?
```ini
[options.packages.find]
include =
    pylint*
```

Reply:
I did not manage to make the exclude works when using this, that's why I added the Manifest.in. Might try harder if you think it's a problem ? :D

Reply:
That's probably a caching issue. Try deleting the `pylint.egg-info`, `build`, and `dist` directories in between `python setup.py sdist bdist_wheel` runs.

Reply:
Ok, I'll try that. Could you review : #4298, I would want to merge it before this one :)

Reply:
Probably the cache as you said 😄